common/rpc/interceptor/namespace_count_limit.go

```diff
@@ -38,7 +39,7 @@ import (
 )
 
 var (
-	ErrNamespaceCountLimitServerBusy = serviceerror.NewResourceExhausted("namespace count limit exceeded")
+	ErrNamespaceCountLimitServerBusy = serviceerror.NewResourceExhausted(enumspb.RESOURCE_EXHAUSTED_CAUSE_CONCURRENT_POLLER_LIMIT, "namespace concurrent poller limit exceeded")
```

Comment:
Because most of the calls come from poller? Is it count limit for # of concurrent weighted requests per namespaces?

Reply:
correct, maybe I should rename the cause as RESOURCE_EXHAUSTED_CAUSE_CONCURRENT_REQUEST_LIMIT?

Reply:
yeah, I guess CONCURRENT_REQUEST_LIMIT makes more sense to me (in case we change the weight for other requests in the future).